build.sbt

```diff
@@ -80,7 +83,8 @@ lazy val db = project
       |  "jdbc:postgresql:gem",
       |  "postgres",
       |  "")
-      |import xa.yolo._
+      |val y = xa.yolo
```

Comment:
It's some convenience syntax for the REPL that lets you run queries and print the results without having to mention which database you're using.